src/CommonDBTM.php

```diff
@@ -337,7 +337,7 @@ public function getFromResultSet($rs)
      *
      * @param DBmysqlIterator $iter Iterator instance
      *
-     * @return CommonDBTM
+     * @return Generator
```

Comment:
The `iterable` pseudo type is more generic.
```suggestion
     * @return iterable
```